config/config-example.yaml

```diff
@@ -36,7 +36,7 @@ processors:
   resourcedetection:
     detectors: [gce]
 
-  # perform custom transformations that aren't supported by the googlemetricstransform processor
+  # perform custom transformations that aren't supported by the metricstransform processor
```

Comment:
[Optional] Do we want to add an example of adding a blank metric label?

Reply:
No, because this is a temporary hack that no-one should use.